spec/index.html

```diff
@@ -117,49 +117,36 @@ <h2>Abstract</h2>
         <a href="https://www.w3.org/groups/wg/rdf-star">RDF Star Working Group</a> as part of the
         update of specifications for format and errata.
       </p>
-    </section>
 
-    <section id="sparql12-documents" class="introductory">
-      <h2>SPARQL 1.2 Documents</h2>
-      <ul>
-        <li><a href="../../sparql-new/spec/index.html">SPARQL 1.2 New</a></li>
-        <li><a href="../../sparql-concepts/spec/index.html">SPARQL 1.2 Overview</a></li>
-        <li><a href="../../sparql-query/spec/index.html">SPARQL 1.2 Query Language</a></li>
-        <li><a href="../../sparql-update/spec/index.html">SPARQL 1.2 Update</a></li>
-        <li><a href="../../sparql-entailment/spec/index.html">SPARQL 1.2 Entailment Regimes</a></li>
-        <li><a href="../../sparql-service-description/spec/index.html">SPARQL 1.2 Service Description</a></li>
-        <li><a href="../../sparql-federated-query/spec/index.html">SPARQL 1.2 Federated Query</a></li>
-        <li><a href="../../sparql-results-json/spec/index.html">SPARQL 1.2 Query Results JSON Format</a></li>
-        <li><a href="../../sparql-results-csv-tsv/spec/index.html">SPARQL 1.2 Query Results CSV and TSV Formats</a></li>
-        <li><a href="../../sparql-results-xml/spec/index.html">SPARQL 1.2 Query Results XML Format</a></li>
-        <li><a href="../../sparql-graph-store-protocol/spec/index.html">SPARQL 1.2 Graph Store Protocol</a></li>
-        <li><a href="../../sparql-protocol/spec/index.html">SPARQL 1.2 Protocol</a></li>
-      </ul>
+      <section id="related" data-include="./common/related.html"></section>
```

Comment:
Separate discussion.

This list is in the initial PR because every SPARQL document had a section linking to the other documents.

Putting all RDF specs as well (before SPARQL) isn't useful IMO.

Reply:
One of the things to discuss. For 1.1 the document set was just for SPARQL. For 1.2, it includes RDF. But, we could certainly split the common/related.html into two parts, and join them as appropriate in the main doc. The group needs to decide on common front-matter.

note that the entire document could be stitched together at runtime from separate parts, with appropriate JavaScript fix ups,

Reply:
Minimally, each document in the set should link to a single document which lists and links to each in the set. That's an extra step for readers to go through. (It's bad enough that links within the text take a reader to the endnotes, in which they must manually find the listing of the desired external reference document (There's no helpful highlight, and what line is at the top of the window depends substantially on the length of the list and the size of the window.), and click again to get there.)